modules/log6x-rn-6.0.1.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+// cluster-logging-release-notes.adoc
+:_mod-docs-content-type: REFERENCE
+[id="cluster-logging-release-notes-6-0-1_{context}"]
+= Logging 6.0.1
+This release includes link:https://access.redhat.com/errata/RHSA-2024:8315[OpenShift Logging Bug Fix Release 6.0.1].
+
+// 4.17
+
+[id="openshift-logging-6-0-1-bug-fixes_{context}"]
+== Bug fixes
+
+* With this update, the default memory limit for the collector has been increased from 1024Mi to 2024Mi. However, users should always adjust their resource limits according to their cluster specifications and needs. (link:https://issues.redhat.com/browse/LOG-6180[LOG-6180])
+
+* Before this update, the Loki Operator failed to add the default `namespace` label to all `AlertingRule` resources, which caused the User-Workload-Monitoring Alertmanager to skip routing these alerts. This update adds the rule namespace as a label to all alerting and recording rules, resolving the issue and restoring proper alert routing in Alertmanager.
```

Comment:
Here, we have `namespace` in backticks because it's referring to the label. That makes sense.

In the following sentence, would it make sense to have `rule namespace` in backticks since it's referring to the label? IS that the label? 

Maybe not. Just food for thought.

Reply:
I checked the code https://github.com/openshift/loki/pull/349/files#diff-f4e0a4fec91b9da0f077301b6d317b5327263058df9dcc18ad327ac0bebe2240 and no, that is not the label. I believe this is correct as written.